src/llmcompressor/modifiers/awq/base.py

```diff
@@ -595,13 +599,12 @@ def _compute_best_scale(
                     )
 
             # W * X
-            int_w_output = self._forward_input_with_kwargs(
-                module=module2inspect, inputs=x, input_kwargs=self._module_kwargs
-            )
-            int_w_output = int_w_output.clip(
-                torch.finfo(int_w_output.dtype).min,
-                torch.finfo(int_w_output.dtype).max,
-            )
+            with HooksMixin.disable_hooks():
+                int_w_output = self._run_samples(parent_module)
+                int_w_output = int_w_output.clip(
```

Comment:
Does this line actually do anything? Isn't `int_w_output` by definition already in between the min and max values of its dtype?